internal/clearingway/menu.go

```diff
@@ -0,0 +1,198 @@
+package clearingway
+
+import (
+	"fmt"
+
+	"github.com/Veraticus/clearingway/internal/discord"
+	"github.com/bwmarrin/discordgo"
+)
+
+// types of UI elements
+type MenuType string
+const(
+	MenuMain      MenuType = "menuMain"
+	MenuVerify    MenuType = "menuVerify"
+	MenuRemove    MenuType = "menuRemove"
+	MenuEncounter MenuType = "menuEncounter"
+)
+
+type CommandType string
+const(
+	CommandMenu CommandType = "menu"
+	CommandSub1 CommandType = "sub1"
+	CommandSub2 CommandType = "sub2"
+	CommandSub3 CommandType = "sub3"
```

Comment:
I could make a command type for each unique function, but that would be a lot of variables for what can be described as a menu's subfunctions so I opted for this instead. If it's good practice to do that I don't have any issue with adding them all as unique command types.

Reply:
since we're going to build on this then i think that this is worth doing. there is probably some categorization we can do with the command types. let's also go ahead and make an enum for the types as well

Reply:
I somehow completely forgot about this, sorry it took so long to get to oof